Src/Extern/Conduit/AMReX_Conduit_Blueprint_ParticlesI.H

```diff
@@ -100,7 +100,7 @@ ParticleTileToBlueprint(const ParticleTile<NStructReal,
     for (int i = 0; i < NStructReal; i++)
     {
         ParticleReal* val = reinterpret_cast<ParticleReal*>(pbuf); pbuf += sizeof(ParticleReal);
-        conduit::Node &n_f = n_fields[real_comp_names[vname_real_idx]];
+        conduit::Node &n_f = n_fields[real_comp_names.at(vname_real_idx)];
```

Comment:
@cyrush thank you for the details.
So the logic part here: is this the place where we create the key in `n_fields` or do we expect the key to already exist at this point?

From what I currently understand, it looks to me like we create the key here for the first time, so I would, contrary to what I thought earlier, not replace it with `fetch_existing`.

Reply:
correct -- this is where we create the key. 
We are using the name from real_comp_names to create a new node in our tree, then populating the children of that node with the rest of the info.